inst/include/sampler.h

```diff
@@ -551,6 +548,21 @@ namespace smc {
         }
     }
     
+	template <class Space>
+	void sampler<Space>::Multinomial(unsigned n, unsigned k, arma::vec w, unsigned int * X) {		
```

Comment:
Yeah I agree it would be better without that helper function. The Multinomial function uses rmultinom but it has a few extra lines to deal with the fact that uRSCount is of type arma::Col<unsigned int> and rmultinom wants int *. I used uRSCount.memptr() to get type unsigned int * but I was having trouble casting to int *. Do you think it's an okay solution to just change uRSCount to type arma::Col<int>? That's what I've tried and I'll do a pull request now for it. Feel free to reject it obviously if it's a bad solution.